RF24.h

```diff
@@ -598,6 +598,46 @@ class RF24 {
      * a `1` means the feature is enabled.
      */
     void printPrettyDetails(void);
+    
+    /**
+     * Put a giant block of debugging information in a char array. This function
+     * differs from printDetails() because it makes the information more
+     * understandable without having to look up the datasheet or convert
+     * hexadecimal to binary. Only use this function if your application can
+     * spare extra bytes of memory.
+     *
+     * @warning use a buffer of sufficient size for debugging_information, start 
+     * with a sizeof >= 1024
+     ^ @param debugging_information The c-string buffer that the debugging 
```

Comment:
1. "sizeof" should be space delimited: "size of"
2. Please use proper English and capitalize the first word of the sentence: "Use a buffer ..." not "use a buffer...". It also wouldn't hurt to have a period at the end.
3. The `^` seems to be a typo again. The line should start with `*`. Just to show why this is important:
![image](https://user-images.githubusercontent.com/14963867/149393811-e516a4c6-4ef6-49e3-a807-afc0745768bd.png)
Notice the typo `^` shows up at the end of "size of >= 1024"